### PR TITLE
[FEAT] Prepare for release

### PR DESCRIPTION
**Pull request by bolismauro**
This PR it has been opened simply to track changes needed to make Tempura work with the [experimental version of katana](https://github.com/BendingSpoons/katana-swift/tree/experimental) we're developing.

It should not be merged until we are sure which are the final APIs of both Katana and Tempura

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ use_frameworks!
 
 target 'Tempura' do
   platform :ios, '9.0'
-  podspec
+  pod 'Katana', :git => 'https://github.com/BendingSpoons/katana-swift', :branch => 'experimental'
```
Comment (cipolleschi) [inline Podfile]: I think you should wait for katana to be released and then update the podfile accordingly.

File: Tempura/Core/ViewController+Containment.swift [hunk 2]
```diff
@@ -32,6 +32,7 @@ extension ViewController {
 /// This is the View that will contain the View of the managed ViewController
 public class ContainerView: UIView {
   
+  /// Layout of the subviews
```
Comment (cipolleschi) [inline Tempura/Core/ViewController+Containment.swift]: Being an overridden method you just have to fallback to the Apple doc with something like 
`- see layoutSubviews()`

File: Podfile (hunk 1 above)
Comment (bolismauro) [reply Podfile]: Done!

File: Tempura/Core/ViewController+Containment.swift (hunk 2 above)
Comment (smaramba) [reply Tempura/Core/ViewController+Containment.swift]: I like it.

**Pull request merged**